packages/browser/src/exports.ts

```diff
@@ -28,6 +28,8 @@ export {
   captureException,
   captureEvent,
   captureMessage,
+  captureUserFeedback,
+  createUserFeedbackEnvelope,
```

Comment:
Do we still need `createUserFeedbackEnvelope` to be exported from browser? 

If not then let's stop exporting it to decrease API surface. If we need it, no worries - also fine.

Reply:
I don't think we need to keep exporting it tbh. But I left the line in to highlight it.

I defer to the SDK team on a) whether we should remove it b) lmk all the places to document the change. LMK!